### ISSUE TITLE
 no ranges for Scorematrix

### ISSUE THREAD

**Issue by Blosberg** [BIMSBbioinfo/pigx_bsseq#142]
I'm still not sure whether this is related to another issue, or a new one on its own, but I've now seen this error here:

```
Quitting from lines 382-421 (index.Rmd) 
Error in genomation::ScoreMatrix(params$bigwigFile, TSS_proximal, strand.aware = TRUE,  : 
  There are no ranges selected
```

**Comment by al2na** [comment]
is TSS_proximal populated ? the error vaguely suggests that TSS_proximal is
either empty or the ranges in it do not overlap with the bigWig file, but
that's just a guess.

**Comment by alexg9010** [comment]
Should be fixed with 35c46b3606aec0ec48b0dd6ae470908dd8ab06f2
